api/Permissions.json

```
@@ -702,7 +703,8 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": "46"
+              "version_added": "46",
+              "notes": "An alias for the `notifications` permission (returns the same value)."
```

```suggestion
              "notes": "Same as the `notifications` permission (returns the same value)."
```

---

Note my comment in https://github.com/mdn/browser-compat-data/pull/24359#issuecomment-2392469301 - though looking at other cases in BCD, this would be more common (
```suggestion
              "notes": "Alias for the `notifications` permission (returns the same value)."
```